app/services/idv/flows/inherited_proofing_flow.rb

```diff
@@ -3,17 +3,34 @@ module Flows
     class InheritedProofingFlow < Flow::BaseFlow
       STEPS = {
         get_started: Idv::Steps::InheritedProofing::GetStartedStep,
+        agreement: Idv::Steps::InheritedProofing::AgreementStep,
+        verify_info: Idv::Steps::InheritedProofing::VerifyInfoStep,
+        verify_phone_or_address: Idv::Steps::InheritedProofing::PhoneStep,
       }.freeze
 
       STEP_INDICATOR_STEPS = [
         { name: :getting_started },
+        { name: :verify_info },
+        { name: :verify_phone_or_address },
```

Comment:
It appears we're verifying address as well as phone (even though address cannot be altered), so I think this is okay; otherwise, `verify_phone` might've been more appropriate.